docs/features/indiscussion/readme.md

```diff
@@ -0,0 +1,6 @@
+New feature spec documents that are still under review live here.
```

Comment:
I don't understand this. If a feature is under discussion, wouldn't that be an open PR where we discuss it? Once we all agree, we can just move it to the `docs/ongoing/` folder.